includes/class-sensei-admin.php

```diff
@@ -397,17 +397,20 @@ public function register_scripts( $hook ) {
 	 * @return bool
 	 */
 	private function has_custom_navigation( $screen ) {
-		$screens_with_custom_navigation = [
-			'edit-course',
-			'edit-course-category',
-			'edit-module',
-			'edit-lesson',
-			'edit-lesson-tag',
-			'edit-question',
-			'edit-question-category',
-			'course_page_' . Sensei_Analysis::PAGE_SLUG,
-			'course_page_sensei_learners',
-		];
+		$screens_with_custom_navigation = apply_filters(
```

Comment:
Could you include a docblock documenting the hook? Also, it would be nice to include the "hooks" label in the PR and add some hook info in the PR description.

Reply:
Thanks for the review @m1r0 :) I've updated the code here https://github.com/Automattic/sensei/pull/5256/commits/913c84aee84f794000b5d3246012035a7a9389a5, can you kindly take a look again? Thanks

Reply:
> can you kindly take a look again

It would be a pleasure! 🙂